src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadoctype/InputMissingJavadocTypeAllowedAnnotations.java

```diff
@@ -0,0 +1,21 @@
+//non-compiled with jkd11: contains dropped packages
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadoctype;
```

Comment:
please explain why javadoc Check might required non-compiled java file.

Reply:
This file was copied from `JavadocTypeCheck` from which the check was split from. Since the old check had, I made the new one have it.
I literally split the check into 2 and duplicated the tests, except for any additions I had to make for pitest.

Reply:
> My thought was if the old check had it, why wouldn't the new one have it too?

old Javadoc checks lacked our attention. As we create new Check (even it is split ), it is better to do them up to our latest quality standards. Non critical items we can postpone to separate issue, if you ok to fix it later, please create new issue.

Reply:
@romani after looking into this more, it is only non-compilable with Java 11 as comment suggests. I believe this is because of the `javax` import. I will make this compilable with Java 11 by creating my own annotation.

Reply:
right, as we need to test exactly `javax.annotation.Generated;`